plugins/modules/dcnm_res_manager.py

```diff
@@ -0,0 +1,1254 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020-2022 Cisco and/or its affiliates.
```

Comment:
Since this is a new module we can set the copyright to just `2022` and extend it later if modified.

Reply:
Done